wp-graphql.php

```diff
@@ -182,3 +184,21 @@ function graphql_init_appsero_telemetry() {
 }
 
 graphql_init_appsero_telemetry();
+
+add_filter(
```

Comment:
Move this to one of our mu-plugin files in `dexerto-site` repo. Do a check for the existence of `areNodesResolved()` and `resetNodesResolved()` to avoid crashes.

Reply:
I've included this change here: https://github.com/Dexerto/dexerto-site/pull/857